js/leo.js

```diff
@@ -257,6 +273,12 @@ function shutdown() {
     ros.close();
 }
 
+function defaultVideoSrc() {
+    if(typeof robot_namespace == 'undefined')
+        video.src = "http://" + robot_hostname + ":8080/stream?topic=/camera/image_raw&type=ros_compressed";
```

Comment:
We can print some warning to the debug console when we were unable to retrieve the robot namespace and we are assuming it to be `/`